cmd/candi/template_configs.go

```diff
@@ -55,13 +67,15 @@ func LoadServiceConfigs(baseCfg *config.Config) (deps dependency.Dependency) {
 			{{if not .RedisDeps}}// {{end}}dependency.SetRedisPool(redisDeps),
 			{{if not .SQLDeps}}// {{end}}dependency.SetSQLDatabase(sqlDeps),
 			{{if not .MongoDeps}}// {{end}}dependency.SetMongoDatabase(mongoDeps),
+			{{if not .ArangoDeps}}// {{end}}dependency.SetArangoDatabase(arangoDeps),
```

Comment:
ini pakein `AddExtended` aja biar arango-nya ga ikut keimport di core candi
```suggestion
			{{if not .ArangoDeps}}// {{end}}dependency.AddExtended("arangodb", arangoDeps),
```